manifest.toml

```diff
@@ -113,6 +109,21 @@ ram.runtime = "50M"
     main.exposed = "UDP"
 
     [resources.permissions]
+    main.allowed = "visitors"
 
     [resources.apt]
-    packages = "build-essential, libirrlicht-dev, cmake, libbz2-dev, libpng-dev, libjpeg-dev, libxxf86vm-dev, libgl1-mesa-dev, libsqlite3-dev, libogg-dev, libvorbis-dev, libopenal-dev, libcurl4-dev, libfreetype6-dev, zlib1g-dev, libgmp-dev, libjsoncpp-dev, libluajit-5.1-dev, libncurses-dev, libzstd-dev"
+    packages = ["build-essential", 
+                "cmake", 
+                "libsqlite3-dev",
+                "libcurl4-gnutls-dev",
```

Comment:
Hmkay so i realize this dependency was added to the list though i'm not sure why x_X, naively I'd say we can remove it

Reply:
I've changed it to flavour explicitly mentioned by [the docs](https://github.com/minetest/minetest/blob/stable-5/doc/compiling/linux.md) as the virtual package `libcurl4-dev` did not resolve properly no `bookworm` [here](https://ci-apps-bookworm-dev.yunohost.org/ci/job/289). We can try reverting as the problem boiled down to `luajit` version in the end 🤷